actionpack/test/dispatch/test_request_test.rb

```diff
@@ -54,9 +54,7 @@ class TestRequestTest < ActiveSupport::TestCase
   end
 
   test "does not complain when Rails.application is nil" do
-    Rails.stubs(:application).returns(nil)
```

Comment:
@kaspth (the same as one example up there) During testing I found out that it's working without this one stubbed. Rails.application is nil by default in this case - without stubbing it. So, better stub it or change the message?